src/subapps/studioLegacy/components/WorkspaceEditorForm.tsx

```diff
@@ -59,7 +59,10 @@ const WorkspaceEditorForm: React.FC<{
           },
         ]}
       >
-        <Input className="ui-workspace-label-input" />
+        <Input
+          className="ui-workspace-label-input"
+          data-testid="workspace-label"
```

Comment:
small recommendation. consider 'data-test-id'.

Reply:
Interestingly, when I change this to `data-test-id` then the `findByTestId` queries in cypress and testing-library stop working. Looks like [they need](https://testing-library.com/docs/queries/bytestid/) the attribute to be spelled without the two dashes.

Reply:
ok if that's the case then go with 'data-testId'